util/util.go

```diff
@@ -317,3 +317,23 @@ func BlockUntilSuccessful(ctx context.Context, c *ethclient.Client, retryable fu
 	b := backoff.WithContext(backoff.WithMaxRetries(backoff.NewConstantBackOff(5*time.Second), 24), ctx)
 	return backoff.Retry(retryable, b)
 }
+
+func WrapDeployedCode(deployedBytecode string, storageBytecode string) string {
+	deployedBytecode = strings.TrimPrefix(deployedBytecode, "0x")
+	storageBytecode = strings.TrimPrefix(storageBytecode, "0x")
```

Comment:
Maybe use `strings.ToLower` also to make sure the bytecode is not formatted with a mix of uppercase and lowercase characters.

Reply:
https://github.com/0xPolygon/polygon-cli/pull/449/commits/893509c8f2585635ec39a1f06435f0ece73cc5fa